src/main/java/codeu/model/data/AboutMeMessage.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+public class AboutMeMessage {
```

Comment:
I think it's fine to go with what you have for the MVP so that you can move on to another project, but I think it's a little funny to have multiple lines like a chat, rather than just a single  profile that can be edited.  But I'd say leave it as it is so you can move on to the open project, unless you're excited to change it.